test/Test/Serokell/Data/Variant/VariantSpec.hs

```diff
@@ -58,5 +60,5 @@ jsonMid = maybe err id . A.decode . A.encode
 bytesFun :: S.Variant -> S.Variant
 bytesFun (S.VarString s) = S.VarBytes right
   where
-     right = either (error . unpack) id $ S.decode s
+     right = either error id $ S.decode s
```

Comment:
`either f id` is `first f`. Btw, looks like a good rule to include into `.hlint.yaml` :thinking:

Reply:
It's not like that here, `first` is not behaving the same as `either`

Reply:
@vrom911 Oh, you're right. Stupid me :man_facepalming: